src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -25,5 +25,6 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     void GlobalAppearance::OnNavigatedTo(const NavigationEventArgs& e)
     {
         _ViewModel = e.Parameter().as<Editor::GlobalAppearanceViewModel>();
+        AlwaysShowTabsToggleSwitch().IsEnabled(!_ViewModel.ShowTabsInTitlebar());

```

Comment:
Now that you have `IsEnabled` in the xaml file, you don't need this line any more ... unless the XAML is not working properly! :)